src/main/java/net/landofrails/landofsignals/configs/LandOfSignalsConfig.java

```diff
@@ -8,4 +8,16 @@
 @SuppressWarnings({"java:S1118", "java:S1444"})
 public class LandOfSignalsConfig {
 
+    @ConfigFile.Comment("Enables preloading, slower startup, less lag spikes. If you want to use it, set it to \"true\". | Default: false")
+    public static boolean preloadModels = false;
+
+    @ConfigFile.Name("experimental")
+    @ConfigFile.Comment("Experimental features - activate at own risk")
+    public static class Experimental {
+
+        @ConfigFile.Comment("Rescale items to slot size - needs \"preloadModels\" to be activated | Default: false")
+        public static boolean rescaleItems = false;
```

Comment:
Why default false?
I think this is a feature that most people would like to have and I think everyone accepts the longer loading time for the added value and if it's really about the loading time you can turn it off again.
```suggestion
        public static boolean rescaleItems = true;
```

Reply:
Why is this not set to true?
1.) This option needs the preloadModels to be set to true
2.) It's not working for all the items available. Some might be distorted by this logic.